src/common/FileSystem.cpp

```diff
@@ -2386,12 +2395,36 @@ bool ParsePakName(const char* begin, const char* end, std::string& name, std::st
 	return true;
 }
 
+std::string MakePakFileName(Str::StringRef name, Str::StringRef version, Util::optional<uint32_t> checksum)
+{
+	if ( version.compare("") != 0 ) {
+		// versioned dpk
+		return Str::Format("%s.%s", MakePakName(name, version, *checksum), "pk3");
```

Comment:
Should "versioned dpk" paks not have the suffix "dpk" and "legacy unversioned pk3" have the suffix "pk3"?